lib/vector-common/src/event_data_eq.rs

```diff
@@ -18,7 +18,12 @@ macro_rules! assert_event_data_eq {
                     left.event_data_eq(right),
                     "assertion failed: {}\n\n{}\n",
                     $message,
-                    pretty_assertions::Comparison::new(left, right),
+                    similar_asserts::SimpleDiff::from_str(
```

Comment:
This is definitely uglier, is there a better way we could compare?

Reply:
This is actually what `pretty_assertions` was [doing already](https://github.com/rust-pretty-assertions/rust-pretty-assertions/blob/27ee62821f4c812e56484c9cc73ff7f27382a02a/pretty_assertions/src/lib.rs#L128-L141).

Reply:
Haha, but I don't have to see it! Obviously not a big thing.